packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -189,6 +202,97 @@ describe("calcite-alert", () => {
     expect(await alert3.isVisible()).toBe(true);
   });
 
+  it("should queue alerts", async () => {
+    const page = await newE2EPage();
+    await skipAnimations(page);
+    await page.setContent(`
+    <div>
```

Comment:
Nitpick: is this container `div` necessary?

Reply:
removed